lua/modules/hyperspace.i

```diff
@@ -446,11 +446,18 @@ playerVariableType playerVariables;
 
 %rename("setWindowTitle") sys_graphics_set_window_title;
 
-/* %rename("%s") ScoreKeeper; */
+%rename("%s") ScoreKeeper;
+%rename("%s") ScoreKeeper::TopScore;
+%rename("%s") ScoreKeeper::TopScore::sector; // returns the current sector as an integer value
+%rename("%s") ScoreKeeper::TopScore::score; // returns the current score as an integer value
```

Comment:
`TopScore` isn't a member of `ScoreKeeper`. You needed to expose `currentScore`, which is itself an instance of `TopScore`, then expose the `TopScore` structure and its `sector` and `score` members.